### ISSUE TITLE
Documentation cleanup

### ISSUE THREAD

**Issue by posix4e** [mozilla/sccache#157]
- [ ] The logging section of the readme still isn't clear. It's not clear to me when logs go to what output device. We need to mention that the logging happens both on the sccache server along with the sccache client. Maybe a summary of a few examples and what would happen would clarify things? I was able to figure it out but the two components each logging is confusing for a user.
- [ ] All of the environment variables should be listed in one clear table, even if it is a duplicate. Examples should be given on how to set them. For instance is the gcs storage path gs://backet_name or does bucket_name work?
- [ ] Something about what features are default and which are optional
- [ ] Maybe mention an irc channel like rust-dev-tools on irc mozilla to get help. People like badges!

**Comment by luser** [comment]
See also https://github.com/mozilla/sccache/issues/151 re: logging.

**Comment by fxb** [comment]
I have been trying to setup sccache to use GCS as storage and I can't get it to work at all. I have verified that the service account has access to the bucket using gcloud and gsutil. Some example configuration would be nice. Maybe @cramertj can add some?

**Comment by cramertj** [comment]
@fxb Here were the env vars I used:
```
SCCACHE_GCS_BUCKET=sccache-test-bucket
SCCACHE_GCS_KEY_PATH=~/Downloads/gce-test-sa.json
SCCACHE_GCS_RW_MODE="READ_WRITE" 
```

**Comment by posix4e** [comment]
@fxb  same place you are.

**Comment by posix4e** [comment]
@cramertj  is that a world bucket?

**Comment by cramertj** [comment]
@posix4e Huh-- I'm retrying it right now and I'm getting the same error on master. I tested it before it got merged, but I must have messed up when I tested the version after [switching from the openssl crate to the pem-parser crate](https://github.com/mozilla/sccache/commit/39fdfd83a84ccf1f1a8df58588c33953bebe702d). It works fine for me when I revert that commit. @luser Do you mind if we revert that commit until I can figure out why the pem-parser version isn't working right?

@posix4e @fxb Try reverting 39fdfd83a84cc and make sure that also works for you.

**Comment by cramertj** [comment]
It's a two-line commit that calls the only function in the pem-parser crate-- I don't have any idea what could be wrong unless the pem-parser crate just doesn't work for private keys.

**Comment by posix4e** [comment]
That revert fixed my gcs bucket. Thanks!

**Comment by Idolf** [comment]
In case anybody else finds this issue and is confused about why GCS does not seem to be working:

I managed to the current master (98d2b30) to work with GCS. I build it using a setup very similar to the one in the `scripts/build-release.sh`, however I used `--features=gcs --no-default-features`.

To actually run it, I needed to use the work-around from japaric/cross#119. Specifically I needed to set `SSL_CERT_DIR=/etc/ssl/certs/`.
